crates/wasmi/src/engine/bytecode2/mod.rs

```diff
@@ -0,0 +1,3895 @@
+#![allow(dead_code)] // TODO: remove
```

Comment:
Why not rename `bytecode2` to `regmach`? The tests part use `regmach` name.

Reply:
This is just a WIP state. The final goal is to entirely replace the old stack machine based `wasmi` executor and bytecode.

Reply:
You means only exist one kind instruction set ?
Not good news for me...

Reply:
Yes, we do not want to support and maintain multiple `wasmi` bytecodes and execution engines.

> Not good news for me...

Why? Can you please elaborate?

Reply:
@yjhmelody I would really appreciate some elaboration.